cime_config/acme/machines/config_batch.xml

```diff
@@ -133,13 +134,28 @@
         <directive>-A {{ PROJECT }}</directive>
         <directive>-l nodes={{ num_nodes }}:ppn={{ tasks_per_node }}</directive>
       </directives>
+      <queues>
+	<queue>shared</queue>
+      </queues>
+      <walltimes>
+	<walltime default="true">15:00</walltime>
+      </walltimes>
     </batch_system>
 
     <!-- edison is SLURM as of Jan-4-2016 -->
     <batch_system MACH="edison" version="x.y">
       <directives>
         <directive> -A {{ account }} </directive>
       </directives>
+      <queues>
```

Comment:
These are moved from config_machines.xml so that all batch information is in one place.